scripts/tfw_lib.sh

```diff
@@ -27,12 +27,55 @@ declare -r TFW_NAME=`basename $0` # program name (comm name in ps)
 declare -r TFW_NETDEV_PATH="/sys/class/net/"
 declare -r TFW_SCRIPTS="$TFW_ROOT/scripts"
 declare -r CPUS_N=$(grep -c processor /proc/cpuinfo)
+declare -r IRQB_CONF_PATH="/etc/default/irqbalance"
+declare -r SYSD_IRQB_PATH="/lib/systemd/system/irqbalance.service"
+declare -r BAN_CONF_VAR="BAN_IRQS"
+declare -a IRQS_GLOB_LIST
 
 calc()
 {
 	echo "$1" | bc -iq | tail -1
 }
 
+# Assigned IRQs can be reassigned by irqbalance. To avoid this, assigned vectors
+# should be added to irqbalance config with '--banirq' option.
+irqbalance_ban_irqs()
```

Comment:
The third-party configuration file is modified here, so we need to add the good comment to document that the `BAN_IRQ` variable is automatically generated by the Tempesta. Also there is no need to modify the systemd service file and call `systemd daemon-reload`. We should only modify `/etc/default/irqbalance` in the following way: 
```
# Generated by Tempesta to restrict rebalancing for NIC irqs.... 
TFW_BAN_IRQS="--banirq ..." 

IRQBALANCE_ARGS="$TFW_BAN_IRQS"
```
And don't forget the quotes around `BAN_IRQS` value.